cli-client/recognizer_stream.py

```diff
@@ -173,6 +174,20 @@ def wait_server(self) -> bool:
         
         return True
     
+    @staticmethod
+    def divide_audio(audio: bytes, chunk_size: int = 20000):
+        audioLength = len(audio)
```

Comment:
It is very common for python variables and methods to use snake case. Maybe it would be a good idea to change this methods variables to snake case "_".